x-pack/plugins/enterprise_search/common/ml_inference_pipeline/index.ts

```diff
@@ -11,6 +11,7 @@ import {
   MlTrainedModelConfig,
   MlTrainedModelStats,
 } from '@elastic/elasticsearch/lib/api/types';
+import { ElasticModels } from '@kbn/ml-plugin/public/application/services/elastic_models_service';
```

Comment:
You should not statically import this service. Instead do the following:
- Add `ml` as a dependency to your plugin 
- If you need to consume it client-side, it is available as the `start` dependency in your plugin. 
- For server-side consumption, from the `setup` use `getELSER` method

Reply:
I see you're already consuming it correctly from the public start. 👍🏻 Please check the suggestion for importing the correct type below.

Reply:
I did completely change the way our code is using the ML plugin as I have been doing some heavy refactoring. Though, I am using the MLPluginStart to retrieve the model.